manifests/overlays/dev/kustomization.yaml

```diff
@@ -11,7 +11,7 @@ commonLabels:
   app.kubernetes.io/component: mpijob
 images:
 - name: mpioperator/mpi-operator
-  newName: gcr.io/acondor-gke-dev/mpi-operator
+  newName: dev-registry/mpi-operator
```

Comment:
Is this change intentional? Does this exist?

Reply:
Neither exist (my GCR is private). It's intended for developers to use their preferred registry.